StripeFinancialConnections/StripeFinancialConnections/Source/FinancialConnectionsSheet.swift

```diff
@@ -143,13 +143,20 @@ final public class FinancialConnectionsSheet {
                     switch completedResult {
                     case .financialConnections(let session):
                         completion(.completed(session: session))
-                    case .instantDebits:
-                        let errorDescription = "Instant Debits is not supported via this interface."
-                        assertionFailure(errorDescription)
+                    case .instantDebits(let linkedBank):
+                        // TODO(mats): Add support for instant debits.
```

Comment:
you might have spoken to Till about this but right now we only support Instant Debits utilizing Financial Connections SDK through the Payment SDK payment sheet

not sure if this project would change that? would be interesting to hear! but I've only launched it through the Payment Sheet example app